packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -162,6 +169,22 @@ int main_( Teuchos::CommandLineProcessor &clp, int argc, char *argv[] )
 
         elapsed_seconds = end - start;
         os << "radius " << elapsed_seconds.count() << "\n";
+
+        auto offset_clone = DataTransferKit::clone( offset );
+        DataTransferKit::adjacentDifference( offset, offset_clone );
+        double const max = DataTransferKit::max( offset_clone );
+        double const avg =
+            DataTransferKit::lastElement( offset ) / ( offset.extent( 0 ) - 1 );
+        double min = max;
+        auto offset_host = Kokkos::create_mirror_view_and_copy(
+            Kokkos::HostSpace(), offset_clone );
+        for ( unsigned int i = 1; i < offset_host.extent( 0 ); ++i )
+            if ( offset_host( i ) < min )
+                min = offset_host( i );
```

Comment:
Add `DataTransferKit::min`?

Reply:
Doesn't work you always get 0 because the first element after an adjacent difference is 0